web/modules/custom/dbcdk_community/src/Plugin/Block/ProfileBlock.php

```diff
@@ -206,6 +222,36 @@ protected function parseProfile(Profile $profile, array $fields) {
   }
 
   /**
+   * Get a Community Profile's Roles.
+   *
+   * @param \DBCDK\CommunityServices\Model\Profile $profile
+   *   The Community Profile.
+   *
+   * @return array $list_items
+   *   Return an array of the Profile's Community Roles.
+   */
+  protected function getProfileRoles(Profile $profile) {
```

Comment:
I am considering where it would make sense to move loading of roles up with the profile load to keep the data load together. However I think this can wait until we get a better approach to handling of profile roles.